nomenclature/__init__.py

```diff
@@ -13,6 +13,7 @@
     RegionProcessor,
     RequiredDataValidator,
 )
+from nomenclature.countries import countries
```

Comment:
I would guess that there's a `# noqa` missing to appease black or flake8
```suggestion
from nomenclature.countries import countries  # noqa
```

Reply:
Done.